Templates/HttpTrigger/function.json

```diff
@@ -2,11 +2,13 @@
   "bindings": [
     {
       "type": "httpTrigger",
-      "direction": "in"
+      "direction": "in",
+      "name": "req"
```

Comment:
We do already default those names, so I could remove this. However, when you add a new binding, we currently always include the name, thus I though it was a good idea our examples should always include the name, as well, even though we have good defaults.